src/wp-includes/block-supports/layout.php

```diff
@@ -308,6 +308,26 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 	return '';
 }
 
+/**
+ * Gets classname from last tag in a string of HTML.
+ *
+ * @since 6.2.0
+ * @access private
+ *
+ * @param string $html markup to be processed.
+ * @return string String of inner wrapper classnames.
+ */
+function wp_get_classnames_from_last_tag( $html ) {
```

Comment:
Obligatory "Naming things is hard", but I wonder if more context in this function name is needed.

Why?

"Tag" could mean one of at least two things when it comes to WordPress:  a taxonomy type or a HTML element.

@hellofromtonya @felixarntz Any thoughts on the naming here?

Reply:
I think the name is okay (WordPress core already uses the word "tag" for both of the two meanings in various function names), but more importantly I think this conversation shouldn't really be relevant given @Mamaduka's feedback that we should not need to introduce this function in the first place.